### ISSUE TITLE
ControlNet training code

### ISSUE THREAD

**Issue by cseander** [PixArt-alpha/PixArt-sigma#111]
I understand if you guys are busy with other things but right now would be the best opportunity to release any tooling for this model as SD3 is such a mess, I strongly believe that the community could pick up PixArt Sigma and make it the new standard.

**Comment by lawrence-cj** [comment]
Definitely. Actually we support the ControlNet in our Alpha version. We will integrate it into Sigma ASAP.

**Comment by cseander** [comment]
Great! I just saw your announcement of a new model and I think the whole community would very much appreciate if you have something similar for that one. Keep up the good work, you're doing awesome stuff!

**Comment by lawrence-cj** [comment]
Thanks for support man.

**Comment by QAQdev** [comment]
@lawrence-cj hello! It seems that Sigma does not support ControlNet training for now. Do you still have the plan to release this part of code? Thanks!
